monosize.config.mjs

```diff
@@ -11,6 +13,13 @@ const config = {
   bundler: webpackBundler(config => {
     return config;
   }),
+  reportResolvers: {
+    packageName: async packageRoot => {
```

Comment:
leveraging package override to get same output as before breaking changes. without this all reports would use `package.json#name`